packages/components/src/components/Drawer/Drawer.test.tsx

```diff
@@ -49,23 +49,7 @@ describe("<Drawer />", () => {
 
     await user.keyboard("{Escape}");
     await waitFor(() => {
-      expect(screen.getByRole("dialog")).not.toBeVisible();
-    });
-  });
-
-  it("should open a drawer and close it by clicking an element in the background", async () => {
```

Comment:
This doesn't work anymore because when click on `presentation` role, it never removes the Drawer from the page. 
I deleted the test because I assume the lib is already testing it and if we could replace it, we would make it similar to the first test, when we click on a button to open/close the component. 


https://github.com/Localitos/pluto/assets/5320963/17b3c225-d080-4d66-aa0c-09490328f115

cc @rssilva